### DISCUSSION TITLE
Resolver plugin

### DISCUSSION THREAD

**Discussion by jdamick** [semihalev/sdns#90] [Q&A] [answered]
Thanks for the project! 

Just curious, but as this project appears based on CoreDNS, why isn't the resolver plugin built as a plugin to CoreDNS instead of a standalone server?

**Comment by semihalev** [comment, marked as answer]
Early versions of sdns was very basic resolver server. After some discussion with @miekg, we decided convert sdns as a CoreDNS resolver plugin. I implemented all infrastructure for CoreDNS. But I didn't have time rebuild everything for CoreDNS resolver plugin. Nowadays this project continue like standalone server. I'm open new ideas about this.

**Reply by crliu3227** [reply]
Is there any progress about this feature?

**Reply by semihalev** [reply]
I'm still working on this. I will create first repo soon. I have some implementation issues.

**Reply by crliu3227** [reply]
Really looking forward for this update
